examples/custom_icon.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from qtpy.QtWidgets import QLabel
+
+import napari
+from napari.qt import compile_qt_svgs
+
+
+icon_path = Path(__file__).parent / 'grin.svg'  # icon svg in this folder
```

Comment:
Do we want to put the icon in some sort of resource folder? I'm ok with it in examples if we can't think of anywhere else, but maybe @jni has another idea?